dev-tools/docker/linux_aarch64_cross_image/Dockerfile

```diff
@@ -83,3 +83,12 @@ RUN \
   cd .. && \
   rm -rf patchelf-0.13.20210805.a949ff2
 
+# Install CMake
+# v3.5 minimum is required
```

Comment:
This comment is wrong: `CMakeLists.txt` says `cmake_minimum_required(VERSION 3.9)`.

Please also update the same comment in the other files where it appears.